lomap/mcs.py

```diff
@@ -422,6 +424,12 @@ def getMapping(moli, molj, hydrogens=False, fname=None, time_out=150):
     ############ MCS BASED RULES ############
 
     # MCSR Rule
+    #sliu 0/17 add function to calculate tanimoto
+    def mtansr(self,):
```

Comment:
OK, so probably just say "This calculates a Tanimoto similarity of ... for .... Currently this code is not used, but it may be in the future," or something along those lines.